gui/dist_view/viewport.h

```diff
@@ -241,6 +241,7 @@ protected slots:
 
 	// single label to be highlighted
 	int highlightLabel;
+    QVector<int> labels;
```

Comment:
Why is highlightLabel still there? "labels" is a too generic name. Call it "highlightLabels" probably?